rancher2-deployment/entrypoint.sh

```diff
@@ -3,30 +3,40 @@
 # When set trace mode is activated. See `help set` in Bash.
 [[ "$RANCHER2_DEPLOYMENT_TRACE" ]] && set -x
 
-# Bash Strict Mode, s. http://redsymbol.net/articles/unofficial-bash-strict-mode/
+# Bash Strict Mode
+# - Exit immediately if a command exits with a non-zero status.
+# - fails if a command fails at any part of a pipeline
+# - Treat unset variables as an error when substituting.
+# - IFS: word splitting is done by <newline> and <tab>, WITHOUT <space>.
+# For more details s. `help set`, `man bash | less -p '   IFS '` and
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
 set -euo pipefail
 IFS=$'\n\t'
 
 ###############################################################################
 ### define/initialize script wide variables
 
+# This programs name without any leading slashes (i.e same as basename).
```

Comment:
I don't get it

Reply:
I added more text which should explain things better